src/services/pathfinding.service.ts

```diff
@@ -129,16 +152,13 @@ class PathFindingService {
    */
   private findReachableConnectorsFromPOI = (
     poi: POI,
-    escalatorDirection: POICategory
+    POICategories: POICategory[]
```

Comment:
Update docs for this function to use POICategories instead of escalatorDirection